config/hugo.config.toml

```diff
@@ -12,7 +12,7 @@ staticdir = "hugo/static"
 theme =  ""
 
 googleAnalytics = ""
-Paginate = 10
+Paginate = 9
```

Comment:
I'd recommend lowercasing this to stick with the casing used in the rest of this file